code/modules/cargo/supplypod.dm

```diff
@@ -57,6 +57,10 @@
 	var/effectShrapnel = FALSE
 	var/shrapnel_type = /obj/projectile/bullet/shrapnel
 	var/shrapnel_magnitude = 3
+	///Whitelist of things that can be put in, used for reverse mode
+	var/list/whitelist = list()
```

Comment:
Can you turn this into a lazylist?

Reply:
what

Reply:
sent you in the discord coder channel what a lazy list is